com.sejda.Desktop.yml

```diff
@@ -0,0 +1,59 @@
+id: com.sejda.Desktop
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '23.08'
+command: sejda-desktop
+separate-locales: false
+
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --socket=pulseaudio
```

Comment:
Why does this need sound?

Reply:
I removed `--socket=pulseaudio`, it is not necessary